analyses/cms-open-data-ttbar/config.json

```diff
@@ -0,0 +1,68 @@
+{
+    "global": {
+        "SERVICEX_IGNORE_CACHE": false,
```

Comment:
I just noticed we're losing out on the comments in the notebook explaining these settings when doing this. JSON itself also doesn't natively support comments, so we should probably use YAML. Here's the YAML version:
```yaml
global:
  SERVICEX_IGNORE_CACHE: false
  AF: coffea_casa
benchmarking:
  CHUNKSIZE: 200000
  AF_NAME: coffea_casa
  SYSTEMATICS: all
  CORES_PER_WORKER: 2
  NUM_CORES: 4
  DISABLE_PROCESSING: false
  IO_FILE_PERCENT: 4.1
  IO_BRANCHES:
    4.1:
      - Jet_pt
      - Jet_eta
      - Jet_phi
      - Jet_btagCSVV2
      - Jet_mass
      - Muon_pt
      - Electron_pt
    15:
      - LHEPdfWeight
      - GenPart_pdgId
      - CorrT1METJet_phi
    25:
      - LHEPdfWeight
      - GenPart_pt
      - GenPart_eta
      - GenPart_pdgId
      - LHEScaleWeight
    50:
      - LHEPdfWeight
      - GenPart_pt
      - GenPart_eta
      - GenPart_phi
      - GenPart_pdgId
      - GenPart_genPartIdxMother
      - GenPart_statusFlags
      - GenPart_mass
      - LHEScaleWeight
      - GenJet_pt
      - GenPart_status
      - LHEPart_eta
      - LHEPart_phi
      - LHEPart_pt
      - GenJet_eta
      - GenJet_phi
      - Jet_eta
      - Jet_phi
      - SoftActivityJet_pt
      - SoftActivityJet_phi
      - SoftActivityJet_eta
      - GenJet_mass
      - Jet_pt
      - Jet_mass
      - LHEPart_mass
      - Jet_qgl
      - Jet_muonSubtrFactor
      - Jet_puIdDisc
```
This switches the strings for the I/O fraction to numbers, can turn them into strings again if this is any problem.

Reply:
This allows for including the original comments again like
```yaml
global:
  SERVICEX_IGNORE_CACHE: false  # ServiceX: ignore cache with repeated queries
```